client_end_user/shared/src/commonMain/kotlin/presentation/composable/PreferredCard.kt

```diff
@@ -23,10 +24,13 @@ import com.beepbeep.designSystem.ui.theme.Theme
 
 @Composable
 fun PreferredCard(
-    onClick: (priceLevel: String) -> Unit,
+    onClickMealCard: (priceLevel: String) -> Unit = {},
+    onClickRideCard: (flag: Int) -> Unit = {},
     painter: Painter,
     title: String,
-    priceLevel: String,
+    isMeal: Boolean,
+    priceLevel: String = "",
+    flag: Int = 0,
```

Comment:
It could be another name than "flag"
"flag " is unreadable
Great refactoring 👏🤍